media/js/mozorg/home/fundraiser.js

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+(function(Mozilla) {
+    'use strict';
+
+    var fundraiser = document.getElementById('fundraiser');
+    var fundraiserClose = document.getElementById('fundraiser-close');
+    var cookieDurationDays = 1;
+    var cookiesOK = typeof Mozilla.Cookies !== 'undefined' && Mozilla.Cookies.enabled();
+    var storageKey = 'fundraiser-jun2019';
```

Comment:
For the sake of being through I'd probably update this. But the code works just fine as-is and the cookies from the last campaign will have already expired so not a blocker.

Reply:
updated :)